tools/fuzz.go

```diff
@@ -32,6 +33,7 @@ var (
 )
 
 func FuzzMarkdownRenderRaw(data []byte) int {
+	setting.AppURL = "http://localhost:3000/"
```

Comment:
out of curiosity, why is this set to "http://localhost:3000/"? is it a fallback value?

Reply:
Google fuzzes the project, and this sets a default value when things are being fuzzed.

Reply:
you can find more details here: https://google.github.io/oss-fuzz/

Reply:
ah - thanks for the explanation and the link @techknowlogick

Reply:
for some reason I confused fuzzing with fuzzy search 🙈 hence the question...